lib/cs_guide/resources/drink.ex

```diff
@@ -11,7 +11,7 @@ defmodule CsGuide.Resources.Drink do
     field(:abv, :float)
     field(:entry_id, :string)
     field(:deleted, :boolean, default: false)
-    field(:description, :string)
+    field(:description, :string, default: "")
```

Comment:
not a fan of this definition for a `:description` as it does not prevent OWASP attacks.
but since it will be replaced by `Fields.description_plaintext_unlimited` shortly. see: https://github.com/dwyl/fields/issues/20
I'm "OK" with merging it so PO can test.